Samples/V1.0Samples/RemoteMediaSamples/IncidentBot/README.md

```diff
@@ -58,6 +58,118 @@ The sample demostrate an incident process workflow. When a incident raised (thro
 
 5.	API references
 
+# Automated VMSS deployment with Github Worflows (Optional)
```

Comment:
Since VMSS is optional, if users follow doc above this line, can they still get a functional incident bot running on Cloud Service without any change (for instance, ignore/delete files which for VMSS)?

Reply:
Yes, they should be able to follow the cloud deployment and have a incident bot sample running with no changes